Makefile

```diff
@@ -2,7 +2,7 @@ SHELL=/usr/bin/env bash -o errexit
 
 .PHONY: help check check-boskos check-core check-services dry-core core dry-services services all
 
-CONTAINER_ENGINE ?= docker
+CONTAINER_ENGINE ?= podman
```

Comment:
If anything, please replace this with a check which of the two exists instead of just breaking this for existing users.